library/Faktory/Prelude.hs

```diff
@@ -12,6 +12,14 @@ import Control.Monad as X
 import Data.Foldable as X
 import Data.Text as X (Text, pack, unpack)
 import Data.Traversable as X
+import GHC.Stack.Types (HasCallStack)
+import GHC.Stack (callStack)
+
+newtype FaktoryException = FaktoryException StringException
+  deriving newtype (Show, Typeable, Exception)
```

Comment:
Actually, before you merge:

> Since GHC 7.10, all types automatically have [Typeable](https://hackage.haskell.org/package/base-4.16.2.0/docs/Data-Typeable.html#t:Typeable) instances derived. This is in contrast to previous releases where [Typeable](https://hackage.haskell.org/package/base-4.16.2.0/docs/Data-Typeable.html#t:Typeable) had to be explicitly derived using the DeriveDataTypeable language extension.

(via https://hackage.haskell.org/package/base-4.16.2.0/docs/Data-Typeable.html)

I noticed this from the `cabal build` warning:

```
library/Faktory/Prelude.hs:19:27: warning: [-Wderiving-typeable]
    • Deriving ‘Typeable’ has no effect: all types now auto-derive Typeable
    • In the newtype declaration for ‘FaktoryException’
   |
19 |   deriving newtype (Show, Typeable, Exception)
   |                           ^^^^^^^^
```

Reply:
OK I'll remove Typeable